emperor/support_files/js/sceneplotview3d.js

```diff
@@ -162,13 +168,86 @@ define([
       [range.min[x], range.min[y], range.max[z]]
     ];
 
+    for (var i = 0; i < 3; i++){
```

Comment:
If you are always doing 3 calls, will it be easier to simply add the 3 lines here?